### PR TITLE
[SDAN-362] - Additional metadata for planning items

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: assets/agenda/utils.js [hunk 1]
```diff
@@ -405,6 +405,25 @@ export function getInternalNotesFromCoverages(item) {
     return internalNotes;
 }
 
+/**
+ * Get list of subjects
+ *
+ * @param {Object} item
+ * @return {Array}
+ */
+export function getSubjects(item) {
```
Comment (petrjasek) [inline assets/agenda/utils.js]: shouldn't this check for duplicates at some point?

Comment (akintolga) [reply assets/agenda/utils.js]: True. I'll add that check.

**Comment by akintolga** [comment]
I have added the uniqe check

File: assets/agenda/utils.js [hunk 2]
```diff
@@ -405,6 +405,25 @@ export function getInternalNotesFromCoverages(item) {
...
+    let subjects = get(item, 'event.subject') || [];
+    const planningItems = get(item, 'planning_items', []);
+    (planningItems || []).forEach(p => {
+        subjects = subjects.concat(p.subject || []);
+        get(p, 'coverages', []).forEach(c => {
+            subjects = subjects.concat(c.subject || []);
+        });
+    });
+
+    return uniqBy(subjects.filter(subject => !isEmpty(subject)), 'qcode');
```
Comment (petrjasek) [inline assets/agenda/utils.js]: I think in newsroom we store subjects with `code` not `qcode`

Comment (akintolga) [reply assets/agenda/utils.js]: Actually, in wire we missed that coversion. Also by checking the `push` again I just realized that we don't need to check every planning item. So I'll change it again.

**Comment by akintolga** [comment]
Updated `push` to handle subject over there

**Pull request merged**